recipes/scicookie/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scicookie" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scicookie-{{ version }}.tar.gz
+  sha256: b1f6c94b9370a8ef21e5591ff091dc931a0dec806bca7245a5741739c190a214
```

Comment:
```suggestion
  sha256: 99a478a6d91cdc6794c183bf252f30195413ccb8e157b3cea745864002f88a3b
```

Reply:
@xmnlab Is there any reason to change this?

Reply:
that is a good question
every time we change the version of the package, we need to go to pypi (if the package is there) and check the new sha256 for that new file https://pypi.org/project/scicookie/#files